src/render_keyhook_camera.c

```diff
@@ -15,13 +15,13 @@
 void	ft_keyhook_rot_cam(int key, t_vec3 *direction, float inc)
 {
 	if (key == XK_Left)
-		*direction = ft_vec3_rotate_y(*direction, inc);
+		*direction = ft_vec3_rotate_y(*direction, inc * 0.1);
```

Comment:
Can we not decrease inc at its source by a decimal point?

Reply:
Then the increment would be too low for movement, since it is also used there. 
For object rotation it's not that noticable, but camera can be way off especially in slower (base) mode.

But I changed the rotation to use degrees instead of radians (by converting) which has a similiar effect. Object rotation is now also slower but acceptable in my opinion